CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ Highlights
   Cubical Agda-specific support code (previously this behaviour was
   part of `--without-K`).
 
+  Since `--cubical-compatible` mode implies that functions should work
+  with the WIP support for [indexed inductive types in Cubical
```

Comment:
WIP -> preliminary
(even though "WIP" is quite common, better not use such abbreviations)